api/tests/Feature/Notifications/TriggerGovernmentExperienceVerifyEmailTest.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+namespace Tests\Feature\Notifications;
+
+use App\Enums\CafEmploymentType;
+use App\Enums\CafForce;
+use App\Enums\CafRank;
+use App\Enums\EmploymentCategory;
+use App\Enums\ExternalRoleSeniority;
+use App\Enums\ExternalSizeOfOrganization;
+use App\Enums\GovPositionType;
+use App\Enums\WorkExperienceGovEmployeeType;
+use App\Models\AwardExperience;
+use App\Models\Classification;
+use App\Models\CommunityInterest;
+use App\Models\Department;
+use App\Models\EducationExperience;
+use App\Models\Notification;
+use App\Models\PersonalExperience;
+use App\Models\User;
+use App\Models\WorkExperience;
+use Database\Seeders\RolePermissionSeeder;
+use Illuminate\Foundation\Testing\Concerns\InteractsWithExceptionHandling;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Nuwave\Lighthouse\Testing\RefreshesSchemaCache;
+use Tests\TestCase;
+use Tests\UsesProtectedGraphqlEndpoint;
+
+use function PHPUnit\Framework\assertEquals;
+
+class TriggerGovernmentExperienceVerifyEmailTest extends TestCase
+{
+    use InteractsWithExceptionHandling;
+    use MakesGraphQLRequests;
+    use RefreshDatabase;
+    use RefreshesSchemaCache;
+    use UsesProtectedGraphqlEndpoint;
+
+    protected $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed(RolePermissionSeeder::class);
+
+        $this->user = User::factory()
+            ->asApplicant()
+            ->create([
+                'work_email_verified_at' => null,
+            ]);
+    }
+
+    // test notification correctly created via mutation
+    // government work experience, indeterminate, null end date
+    public function testGovernmentWorkExperienceVerifyWorkEmailNotificationMutationSuccessful(): void
+    {
+        $classification = Classification::factory()->create();
+        $department = Department::factory()->create();
+
+        Notification::truncate();
+        assertEquals(0, count(Notification::all()));
```

Comment:
Is it possible to add this to the setup so we don't create new data for each test?

Reply:
So like so, shifting it up in most cases https://github.com/GCTC-NTGC/gc-digital-talent/pull/12737/commits/b5647d2119d6d5800ecc6eceaf81d9a5a2c3bbf8 ?

Reply:
Ya and the classification and department! Sorry I should of been more clear.

Reply:
Ah, okay, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12737/commits/fcdf365eeefbf3ae07268e5b9f78e8c60c2d794f